policy/modules/services/container.te

```diff
@@ -0,0 +1,880 @@
+allow container_domain self:msgq create_msgq_perms;
+allow container_domain self:unix_stream_socket { create_stream_socket_perms connectto };
+allow container_domain self:unix_dgram_socket { sendto create_socket_perms };
+
+manage_dirs_pattern(container_domain, container_file_t, container_file_t)
+manage_files_pattern(container_domain, container_file_t, container_file_t)
+manage_lnk_files_pattern(container_domain, container_file_t, container_file_t)
+manage_sock_files_pattern(container_domain, container_file_t, container_file_t)
+manage_fifo_files_pattern(container_domain, container_file_t, container_file_t)
+rw_chr_files_pattern(container_domain, container_file_t, container_file_t)
+rw_blk_files_pattern(container_domain, container_file_t, container_file_t)
+
+allow container_domain container_ro_file_t:file read_file_perms;
+
+allow container_domain container_file_t:dir mounton;
+allow container_domain container_file_t:filesystem getattr;
+
+can_exec(container_domain, container_file_t)
+
+kernel_getattr_proc(container_domain)
+kernel_list_all_proc(container_domain)
+kernel_read_kernel_sysctls(container_domain)
+kernel_rw_net_sysctls(container_domain)
+kernel_read_system_state(container_domain)
+kernel_dontaudit_search_kernel_sysctl(container_domain)
+
+# corecmd_exec_all_executables(container_domain)
+
+files_dontaudit_getattr_all_dirs(container_domain)
+files_dontaudit_getattr_all_files(container_domain)
+files_dontaudit_getattr_all_symlinks(container_domain)
+files_dontaudit_getattr_all_pipes(container_domain)
+files_dontaudit_getattr_all_sockets(container_domain)
+files_dontaudit_list_all_mountpoints(container_domain)
+files_dontaudit_write_etc_runtime_files(container_domain)
+# files_entrypoint_all_files(container_domain)
+# files_list_var(container_domain)
+# files_list_var_lib(container_domain)
+# files_search_all(container_domain)
+# files_read_config_files(container_domain)
+# files_read_usr_files(container_domain)
+# files_read_usr_symlinks(container_domain)
+
+fs_getattr_all_fs(container_domain)
+# fs_list_inotifyfs(container_domain)
+# for rootless containers
+fs_manage_fusefs_dirs(container_domain)
+fs_manage_fusefs_files(container_domain)
+fs_manage_fusefs_symlinks(container_domain)
+fs_exec_fusefs_files(container_domain)
+fs_fusefs_entry_type(container_domain)
+
+# fs_rw_inherited_tmpfs_files(container_domain)
+# fs_rw_inherited_cifs_files(container_domain)
+# fs_rw_inherited_noxattr_fs_files(container_domain)
+
+auth_dontaudit_read_login_records(container_domain)
+auth_dontaudit_write_login_records(container_domain)
+# auth_search_pam_console_data(container_domain)
+
+# clock_read_adjtime(container_domain)
+
+# init_read_utmp(container_domain)
+# init_dontaudit_write_utmp(container_domain)
+
+libs_dontaudit_setattr_lib_files(container_domain)
+
+# miscfiles_read_localization(container_domain)
+miscfiles_dontaudit_setattr_fonts_cache_dirs(container_domain)
+# miscfiles_read_fonts(container_domain)
```

Comment:
Why all the commented lines? It seems that some of them would be fine, like `miscfiles_read_localization()`

Reply:
Some of these commented lines are leftovers from `virt.te`. The ones I commented I wasn't super sure of. We could remove them since I found them not to be required accesses (and also found them not granted in container-selinux either).

For the more harmless ones like `miscfiles_read_localization` we can leave those in place if you think so.

Reply:
Please see [`9edfc4e` (#434)](https://github.com/SELinuxProject/refpolicy/pull/434/commits/9edfc4eb977b14283add0bab9797641bbbd5b586#diff-356defa60a3e74fb68b6f1c76d0c9be541b9b1ab4a21051c77a519165d3428aaL1131-L1144) to see the lines that were originally commented out in `virt.te`.

Reply:
I'd say definitely drop the ones that were commented in `virt.te`. The rules you found which weren't required we should probably keep in case they're needed for LXC, unless it's a big access.